CHANGELOG

```diff
@@ -39,6 +39,9 @@ Miscellaneous
   the application level.  Example usage of this new interface can be found
   in the examples/sundials directory.
 
+- Added support for output in the Paraview XML format. See Example 5/5p and the
+  new ParaviewDataCollection class.
+
```

Comment:
It Looks like the proper styling is "ParaView" not "Paraview", https://www.paraview.org/. 

Should we update the classes and comments? 

(We use VisIt for https://visit.llnl.gov)